lib/discourse_theme/scaffold.rb

```diff
@@ -105,56 +23,60 @@ def self.generate(dir)
+
+      FileUtils.cp_r(SKELETON_DIR, dir)
+
       Dir.chdir dir do
-        author =
-          loop do
-            input = UI.ask("Who is authoring the theme?", default: ENV["USER"]).to_s.strip
-            if input.empty?
-              UI.error("Author cannot be empty")
-            else
-              break input
-            end
-          end
+        FileUtils.rm_rf(".git")
 
+        author = UI.ask("Who is authoring the theme?", default: "Discourse").to_s.strip
         description = UI.ask("How would you describe this theme?").to_s.strip
 
-        about_template = ABOUT_JSON.dup
-        about_template[:name] = name
-        if is_component
-          about_template[:component] = true
-        else
-          about_template[:color_schemes] = {}
+        about = JSON.parse(File.read("about.json"))
+        about["name"] = name
+        about["authors"] = author
+        if !is_component
+          about.delete("component")
+          about["color_schemes"] = {}
         end
+        File.write("about.json", JSON.pretty_generate(about))
 
-        encoded_name = name.downcase.gsub(/[^a-zA-Z0-9_-]+/, "_")
+        if author != "Discourse"
+          license = File.read("LICENSE")
+          license.sub!(/^(Copyright\s\(c\))\s(.+)/, "\\1 #{author}")
+          File.write("LICENSE", license)
+        end
 
-        write("about.json", JSON.pretty_generate(about_template))
-        write("HELP", HELP)
-        write("LICENSE", LICENSE.sub("#YEAR", "#{Date.today.year}").sub("#AUTHOR", author))
-        write(".eslintrc", ESLINT_RC)
-        write(".gitignore", GIT_IGNORE)
-        write(".template-lintrc.js", TEMPLATE_LINT_RC)
-        write("package.json", PACKAGE_JSON.sub("#AUTHOR", author))
-        write("settings.yml", SETTINGS_YML)
-        write("common/common.scss", "")
-        write("javascripts/discourse/api-initializers/#{encoded_name}.js", API_INITIALIZER)
-        write("locales/en.yml", EN_YML.sub("#DESCRIPTION", description))
+        readme = File.read("README.md")
+        readme.sub!("**Theme Name**", "**#{name}**")
+        File.write("README.md", readme)
+
+        encoded_name = name.downcase.gsub(/[^a-zA-Z0-9_-]+/, "-")
+        FileUtils.mv(
+          "javascripts/discourse/api-initializers/todo.js",
+          "javascripts/discourse/api-initializers/#{encoded_name}.js",
+        )
+
+        i18n = YAML.safe_load(File.read("locales/en.yml"))
+        i18n["en"]["theme_metadata"]["description"] = description
+        File.write("locales/en.yml", YAML.safe_dump(i18n).sub(/\A---\n/, ""))
 
         UI.info "Initializing git repo"
+        FileUtils.rm_rf(".git")
         puts `git init && git symbolic-ref HEAD refs/heads/main`
 
         UI.info "Installing dependencies"
         puts `yarn`
       end
-    end
-
-    private
 
-    def self.write(filename, contents)
-      UI.info "Creating #{filename}"
-      FileUtils.mkdir_p(File.dirname(filename))
-      File.write(filename, contents)
+      puts "✅ Done!"
```

Comment:
IMO after installing dependencies we should automatically create an initial commit with the essential files. Stuff like the api-initializer can be left unstaged.

Bonus points for ascii art ;)

<img width="200" alt="SCR-20240216-ipef" src="https://github.com/discourse/discourse_theme/assets/6270921/23bba6f3-5db8-48d6-94c3-4e2c4e11b765">